.changes/unreleased/Under the Hood-20221207-151813.yaml

```diff
@@ -0,0 +1,8 @@
+kind: Under the Hood
```

Comment:
Probably doesn't need a changelog entry, since it's not a change to code we actually release/distribute — but it's nbd